com.gitlab.librebob.Athenaeum.json

```diff
@@ -0,0 +1,138 @@
+                            ],
+                            "dest-filename": "configure"
+                        }
+                    ],
+                    "cleanup" : [
+                        "/bin",
+                        "/include"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "python3-peewee",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} peewee"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/c9/e4/f82bd273a7c88e7a2f71652324039c7e382f2f88eced01b9799d6c936ad2/peewee-3.9.3.tar.gz",
+                    "sha256": "603337153536b85c3c5b5e84df4ae0f9cbaff09163a3739451d167e0ad3fe1f7"
+                }
+            ]
+        },
+        {
+            "name": "python-setuptools_scm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://files.pythonhosted.org/packages/09/b4/d148a70543b42ff3d81d57381f33104f32b91f970ad7873f463e75bf7453/setuptools_scm-3.1.0.tar.gz",
+                "sha256": "1191f2a136b5e86f7ca8ab00a97ef7aef997131f1f6d4971be69a1ef387d8b40"
+                },
+                {
+                "type": "shell",
+                "commands": [
+                    "sed -i 's/if not platform_/if not dirs/' setup.py"
+                ]
+                }
+            ]
+        },
+        {
+            "name": "python3-python-dateutil",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://files.pythonhosted.org/packages/0e/01/68747933e8d12263d41ce08119620d9a7e5eb72c876a3442257f74490da0/python-dateutil-2.7.5.tar.gz",
+                "sha256": "88f9287c0174266bb0d8cedd395cfba9c58e87e5ad86b2ce58859bc11be3cf02"
+                },
+                {
+                "type": "shell",
+                "commands": [
+                    "sed -i 's/if not platform_/if not dirs/' setup.py"
+                ]
+                }
+            ]
+        },
+        {
+            "name": "athenaeum",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/librebob/athenaeum.git"
```

Comment:
This needs to be a specific release or commit, see the requirements here: https://github.com/flathub/flathub/wiki/App-Requirements